build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -27,6 +27,12 @@ function pre_download()
 	cp -p ${BIN_PATH}/km0_boot_all.bin ${IMG_TOOL_PATH}/km0_boot_all.bin
 	cp -p ${BIN_PATH}/km4_boot_all.bin ${IMG_TOOL_PATH}/km4_boot_all.bin
 	cp -p ${BIN_PATH}/${KERNEL_BIN_NAME} ${IMG_TOOL_PATH}/${KERNEL_BIN_NAME}
+	cp -p ${BIN_PATH}/micom ${IMG_TOOL_PATH}/micom
+	cp -p ${BIN_PATH}/wifi ${IMG_TOOL_PATH}/wifi
+	if test -f ${BIN_PATH}/common_binary; then
+		cp -p ${BIN_PATH}/common_binary ${IMG_TOOL_PATH}/common_binary
+	fi
+
 	if test -f "${SMARTFS_BIN_PATH}"; then
```

Comment:
Is there any case we should flash the smartfs image even though app binaries are based on mtd?

Reply:
As you said, we don't need to flash smartfs image by default. I will remove this.